tests/test-pwm.js

```diff
@@ -132,7 +132,7 @@ setTimeout(function() {
            }
        }
     }, 10);
-}, 1000);
+}, 1040);
```

Comment:
The msTimer is detection pulse, so we must Keep enough time to check it.
       

>  1000=50X20 < total-time < 50X21=1050

I have got 1040 for robustness of this case.